examples/creatives/banner_nonlinear.html

```diff
@@ -0,0 +1,53 @@
+<html>
+  <head>
+    <script src="../simid_protocol.js"> </script>
+    <script src="base_simid_creative.js"> </script>
+    <script src="banner_nonlinear.js"> </script>
+  </head>
+  <style>
+    #close_ad {
+      color: red;
+      border: solid white 1px;
+      font-size: 14;
+      float: right;
+    }
+    #minimize_ad {
+      color:rgb(78, 78, 78);
+      border: solid white 1px;
+      font-size: 14;
+      float: right;
+    }
+    button {
+      cursor: pointer;
+    }
+    body {
+      background: rgba(0,0,0,0.5);
+      width: auto;
+      text-align: center;
+      transition: opacity .5s cubic-bezier(0.0,0.0,0.2,1);
+      text-shadow: 0 0 4px rgba(0,0,0,0.75);
+      cursor: pointer;
+    }
+    body:hover {
+      background: rgba(0,0,0,0.7);
+    }
+    #ad_text {
+      color: white;
+      text-align: center;
+    }
+  </style>
+  <body>
+    <button id="close_ad">X</button>
+    <button id="minimize_ad">_</button>
+    <div id="content_container">
+      <h1 id="ad_text">This is a banner ad</h1>
+    </div>
+    <script>
+      //The creative should first call ready
```

Comment:
Change this comment to  
// This implementation is immediately ready but other creatives may wish to wait for assets to load before calling ready.
as Ryan mentioned on my PR

Reply:
Reordered the comments